source/broadband/webpa_internal.c

```diff
@@ -1130,8 +1130,11 @@ static void checkComponentHealthStatus(char * compName, char * dbusPath, char *s
 	
 	sprintf(tmp,"%s.%s",compName, "Health");
 	parameterNames[0] = tmp;
-	      
+#if defined(RDKB_EMU)	      
+	walStrncpy(l_Subsystem, "",sizeof(l_Subsystem));
```

Comment:
Change this to keep it consistent as used in other parts of the code
#if !defined(RDKB_EMU)
	strncpy(l_Subsystem, "eRT.",sizeof(l_Subsystem));
#endif